src/BloomBrowserUI/lib/long-press/jquery.longpress.js

```diff
@@ -233,8 +233,12 @@
         if (isTextArea()) {
             setTextAreaCaretPosition(activeElement, textAreaCaretPosition);
         } else {
-            EditableDivUtils.makeSelectionIn(activeElement, storedOffset, null, true);
-            setFocusDelayed();
```

Comment:
(a) Can you determine (e.g. with blame?) why someone once thought it was a good thing to delay setting focus, and make sure we haven't broken whatever they were fixing?
(b) Looks like we could now delete the setFocusDelayed method, too. This seems to be the only caller.